_docs/guides/forseti-security-best-practices.md

```diff
@@ -0,0 +1,75 @@
+---
+title: Forseti Security Best Practices 
+order: 004
+---
+# {{ page.title }}
+
+## Service Accounts
+When you use multiple service accounts with your Forseti Security deployments,
+you fulfill the security best practice of least privilege. Following are the
+scenarios for which it's best to use separate service accounts:
+
+ * **Forseti Security Service Account** (required): Used by all core modules of
+the program to provide basic inventory, scanning, and enforcement actions.
```

Comment:
In addition to the naming, would include "GCP" in the description here.  Perhaps:
```provide basic inventory, scanning, and enforcement actions in GCP.```